src/ToolBox/SOS/Strike/platformspecific.h

```diff
@@ -190,6 +190,6 @@ struct DT_RTL_USER_PROCESS_PARAMETERS
 
 #endif // !FEATURE_PAL
 
-#define DT_OS_PAGE_SIZE   4096
+#define DT_OS_PAGE_SIZE   0x10000
 
```

Comment:
There may be complications here for remote debugging with target having separate size as host.

Also this may need a different mechanism to determine the os page size dynamically.